### DISCUSSION TITLE
postgres数据库在v3版本中无法使用的相关问题

### DISCUSSION THREAD

**Discussion by meowluna** [AlistGo/alist#2649] [Q&A]
无报错，死循环，已确认数据库用户名、密码没有问题，换到mysq的数据库l就可以运行
![image](https://user-images.githubusercontent.com/62191675/206456350-bb1c3dd4-b342-4e6f-9cef-f76aac8d8bd6.png)
![image](https://user-images.githubusercontent.com/62191675/206455994-e7bcfddb-2ef4-49aa-9590-6501aa81f13e.png)
![image](https://user-images.githubusercontent.com/62191675/206456331-70bf3609-0d21-4a14-a871-c6f4eeb91291.png)

**Comment by Sodiseng007** [comment]
mysql的数据库也是错误了， 导致网站都打不开了，状态异常
